pkg/cli/application/get.go

```diff
@@ -0,0 +1,68 @@
+package application
+
+import (
+	"context"
+	"os"
+
+	applicationc "github.com/fuseml/fuseml-core/gen/http/application/client"
+	"github.com/fuseml/fuseml-core/pkg/cli/common"
+	"github.com/spf13/cobra"
+)
+
+// GetOptions holds the options for 'application get' sub command
+type GetOptions struct {
+	common.Clients
+	global  *common.GlobalOptions
+	format  *common.FormattingOptions
+	Name    string
+}
+
+// NewGetOptions creates a ApplicationGetOptions struct
+func NewGetOptions(o *common.GlobalOptions) *GetOptions {
+	res := &GetOptions{global: o}
+	res.format = common.NewSingleValueFormattingOptions()
+	return res
+}
+
+// NewSubCmdApplicationGet creates and returns the cobra command for the `application get` CLI command
+func NewSubCmdApplicationGet(gOpt *common.GlobalOptions) *cobra.Command {
+
+	o := NewGetOptions(gOpt)
+
+	cmd := &cobra.Command{
+		Use:   `get {-n|--name NAME}`,
+		Short: "Get an application.",
```

Comment:
When referring to an application sometimes I see you using `application` and other times `Application`, better choose one and stick with it.

Reply:
ok